Src/CSharpier/SyntaxPrinter/ConstraintClauses.cs

```diff
@@ -32,11 +16,6 @@ public static Doc Print(IEnumerable<TypeParameterConstraintClauseSyntax> constra
             constraintClausesList.Select(TypeParameterConstraintClause.Print)
         );
 
-        return Doc.Group(
-            Doc.Indent(groupId != null ? Doc.IfBreak(" ", prefix, groupId) : prefix),
-            groupId != null
-              ? Doc.IfBreak(Doc.Align(2, body), Doc.Indent(body), groupId)
```

Comment:
Heh, funny we never extracted the `2` to a constant. I was kind of hoping this change (on the code side) would be a simple `HalfIndent = DefaultIndent` :smirk: